.github/workflows/test.yml

```diff
@@ -31,9 +31,9 @@ jobs:
           secret-key: ${{ secrets.OP_SECRET_KEY }}
           master-password: ${{ secrets.OP_MASTER_PASSWORD }}
           items: |
-            GitHub Action Test Vault > Test Login
-            GitHub Action Test Vault > Test Password | p12
-            GitHub Action Test Vault > Test Document
+            1Password Action Tests > Test Login
+            1Password Action Tests > Test Password | p12
+            1Password Action Tests > Test Document
       - name: Verify
```

Comment:
Is there a way we can verify that this works in this step? I'm not sure exactly what the command does but it might be something like removing the contents of $XDG_CONFIG_HOME/op, which we could verify.

Reply:
I tried looking at a few directories inside of docker and nothing changed whether or not I signed out or not.  They very well could be hidden, but nothing came out saying this was a session value

![image](https://user-images.githubusercontent.com/1119565/101119681-7e564a80-35b1-11eb-8c28-d3c2afe6e855.png)